iOS/UI/Manga/MangaViewController.swift

```diff
@@ -1160,6 +1160,21 @@ extension MangaViewController: MangaDetailHeaderViewDelegate {
         present(SFSafariViewController(url: url), animated: true)
     }
 
+    // copy manga link when holding down the web view button
+    func safariHeld() {
+        guard
+            let url = manga.url,
+            url.scheme == "https" || url.scheme == "http"
+        else { return }
+        UIPasteboard.general.string = url.absoluteString
+        let alert =
+        UIAlertController(title: NSLocalizedString("LINK_COPIED", comment: ""), message:
+                            NSLocalizedString("LINK_COPIED_TEXT",
+                                              comment: ""), preferredStyle: .alert)
+                    alert.addAction(UIAlertAction(title: "OK", style: .default, handler: nil))
+                    present(alert, animated: true, completion: nil)
```

Comment:
indentation here is a little funky, and completion is nil by default
```suggestion
        let alert = UIAlertController(
            title: NSLocalizedString("LINK_COPIED", comment: ""),
            message: NSLocalizedString("LINK_COPIED_TEXT", comment: ""),
            preferredStyle: .alert
        )
        alert.addAction(UIAlertAction(title: "OK", style: .default, handler: nil))
        present(alert, animated: true)
```